### DISCUSSION TITLE
[声明]打击违法犯罪行为

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#633] [其他]
![1723215836510](https://github.com/user-attachments/assets/380cd543-43f6-4c5a-a199-fb05e32a237b)
近期空洞内诈骗，谋杀以及绑架事件甚多，治安局从不鼓励擅自进行空洞内行为，各位绳匠还请为自己的生命安全着想。钱可以赚无数回，生命只有一次。

**Comment by Tragedysaver1024** [comment]
南无三，面对被治安局渗透一干二净的绳网，佛陀也不禁闭上了眼睛……

**Comment by GUILAISHIXV** [comment]
治安局也渗透到绳网了吗

**Comment by lllliufeng** [comment]
清汤大老爷

**Comment by BITE-7274** [comment]
可恶的治安官
干扰我的事业已经许多次了
下次我要把你拐进黑洞里让你一去不复返：）

**Reply by ZhuYuan-148** [reply]
好哦，这位绳匠很勇嘛
